### ISSUE TITLE
bug: Transforms API references a Transformers Guide that doesn't exist

### ISSUE THREAD

**Issue by ProVega** [bentoml/BentoML#4541]
### Describe the bug

I am new to Bento ML and trying to learn how to deploy a Hugging Face Transformer. I see various example and I want to learn more about the transformers.pipeline method (ex: What is task). I find this page:

https://github.com/bentoml/BentoML/blob/main/docs/source/reference/frameworks/transformers.rst
https://docs.bentoml.com/en/latest/reference/frameworks/transformers.html

But it just says this "Please refer to Transformers guide for more information about how to use Hugging Face Transformers in BentoML." 

Under Guides there is no such article and there is no link. 

### Environment

N/A

**Comment by cserpellZF** [comment]
I don't think it really wants to point to huggingface's page, I think in fact there is a framework documentation missing in BentoML doc.
